src/util.ts

```diff
@@ -1,3 +1,26 @@
-export const withoutLeadingSlash = (string: string) => string.startsWith('/') ? string.substring(1) : string;
```

Comment:
Don't we still need withoutLeadingSlash? It's [used in gatsby-node.ts](https://github.com/jariz/gatsby-plugin-s3/blob/a0917f30df83cd8f1f156207fe9f1a17256fe775/src/gatsby-node.ts#L19).

Reply:
We do. Slipped when I rebased master on my branch. Fixed